contracts/Turnstone.vy

```diff
@@ -103,7 +103,7 @@ def make_checkpoint(valset: Valset) -> bytes32:
 # valset: new validator set to update with
 # consensus: current validator set and signatures
 @external
-def update_valset(new_valset: Valset, consensus: Consensus):
+def update_valset(consensus: Consensus, new_valset: Valset):
```

Comment:
could you rebase with master because this change https://github.com/palomachain/turnstone-evm/blob/master/contracts/Turnstone.vy#L107 is already applied